homeassistant/components/adaptive_lighting/strings.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "Adaptive Lighting",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Choose a name for the Adaptive Lighting",
+        "description": "Every instance can contain multiple lights!",
+        "data": {
+          "name": "[%key:common::config_flow::data::name%]"
+        }
+      }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "Adaptive Lighting options",
+        "description": "All settings for a Adaptive Lighting component. The option names correspond with the YAML settings. No options are shown if you have this entry defined in YAML.",
+        "data": {
+          "lights": "lights",
+          "adapt_brightness": "adapt_brightness",
+          "adapt_color_temp": "adapt_color_temp, adapt color temperature using 'color_temp' if supported",
+          "adapt_rgb_color": "adapt_rgb_color, adapt color temperature using RGB/XY if supported",
+          "initial_transition": "initial_transition, when lights go 'off' to 'on' or when 'sleep_state' changes",
```

Comment:
```suggestion
          "initial_transition": "initial_transition, when lights go 'off' to 'on'",
```

Reply:
Actually, when the sleep_mode switch toggles this is the transition too.

Reply:
Sleep mode is still there? Thought it was removed. What does it do?

Reply:
A switch `switch.adaptive_lighting_sleep_mode_NAMEHERE` is added. When active the lights turn to `sleep_brightness` and `sleep_color_temp`.

Reply:
Oh, didn't see that. Mind doing a sync with the custom HACS repo?

Reply:
Done!